cmd/helpers.go

```diff
@@ -77,8 +102,23 @@ func BuildApplication(binaryName string, forceRebuild bool, buildMode string, pa
 	packSpinner.SetSpinSpeed(50)
 	packSpinner.Start()
 
+	// embed resources
+	targetFiles, err := EmbedAssets()
+	if err != nil {
+		return err
```

Comment:
Does this trigger logging somewhere higher up if it fails?

Reply:
yes, its sent back to the calling cmd command and displayed on console as a build error